deps/ncrypto/ncrypto.cc

```diff
@@ -346,7 +346,7 @@ bool BignumPointer::generate(const PrimeConfig& params,
                              PrimeCheckCallback innerCb) const {
   // BN_generate_prime_ex() calls RAND_bytes_ex() internally.
   // Make sure the CSPRNG is properly seeded.
-  CSPRNG(nullptr, 0);
+  std::ignore = CSPRNG(nullptr, 0);
```

Comment:
```suggestion
  USE(CSPRNG(nullptr, 0));
```

Reply:
The `USE` macro is defined in `src/util.h`, I could also define it in `ncrypto.cc` if you prefer it.

Reply:
Ah, let's use std::ignore than. sorry for the trouble!